### PR TITLE
Support for using patterns update

### PR DESCRIPTION
**Pull request by wiiitek**

### CODE REVIEW

File: report/src/main/webapp/app/layout/toolbar/toolbarBottom.controller.js [hunk 1]
```diff
@@ -17,12 +17,20 @@
...
+		'patternsService', 'metadataAccessService', 'notesService', 'viewModeService', 'suiteInfoService',
+		ToolbarBottomController];
 
-	function ToolbarBottomController($scope, $rootScope, $uibModal, $stateParams, patternsService, metadataAccessService, notesService, viewModeService) {
+	function ToolbarBottomController($scope, $rootScope, $uibModal, $stateParams,
+			patternsService, metadataAccessService, notesService, viewModeService, suiteInfoService) {
 		var vm = this;
 
 		vm.showAcceptButton = patternsMayBeUpdated;
+		// disables accept button if compared against another suite patterns
```
Comment (Skejven) [inline report/src/main/webapp/app/layout/toolbar/toolbarBottom.controller.js]: Duplicated comment.

File: report/src/main/webapp/app/layout/toolbar/toolbarBottom.view.html [hunk 2]
```diff
@@ -57,18 +57,28 @@ <h4 class="toolbar-link-header ellipsis">
...
+                  data-toggle="popover"
+                  data-content="May not be accepted because compared against another suite patterns."
+                  data-html="true"
+                  data-trigger="hover"></i>
+                Accept {{toolbarBottom.viewMode}}
+            </span>
+
             <span class="button js-rebase-suite button-blue"
-                  data-ng-if="toolbarBottom.showAcceptButton()"
+                  data-ng-if="toolbarBottom.showAcceptButton() && !toolbarBottom.acceptButtonDisabled"
```
Comment (Skejven) [inline report/src/main/webapp/app/layout/toolbar/toolbarBottom.view.html]: Maybe we should put `!toolbarBottom.acceptButtonDisabled` logic inside `toolbarBottom.showAcceptButton()`?
And the same below for `toolbarBottom.showRevertButton()`.

File: report/src/main/webapp/app/layout/toolbar/toolbarTop.view.html [hunk 3]
```diff
@@ -23,8 +23,16 @@
...
-        <div class="toolbar-block important-message" ng-if="toolbarTop.patternSuite">
-            This suite comparison is based on <a href="{{toolbarTop.patternSuite.url}}">{{toolbarTop.patternSuite.name}}</a> suite patterns.
+        <div class="toolbar-block" ng-class="{'hidden': toolbarTop.pattern}">
+            <span class="preformatted" data-toggle="popover" data-content="Correlation id" data-trigger="hover" data-container="body">
+                {{toolbarTop.suiteInfo.correlationId}}
+            </span>
+        </div>
+        <div class="toolbar-block important-message" ng-if="toolbarTop.pattern">
+            Uses patterns from <a href="{{toolbarTop.pattern.url}}">{{toolbarTop.pattern.name}}</a>.
+            <a href="https://github.com/Cognifide/aet/wiki/SharedPatterns" target="_blank">
```
Comment (Skejven) [inline report/src/main/webapp/app/layout/toolbar/toolbarTop.view.html]: 👍 great idea

File: report/src/main/webapp/app/layout/toolbar/toolbarBottom.view.html (hunk 2 above)
Comment (wiiitek) [inline report/src/main/webapp/app/layout/toolbar/toolbarBottom.view.html]: done

**Pull request merged**